release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -1128,6 +1128,10 @@ See link:https://access.redhat.com/articles/6955985[Navigating Kubernetes API de
 [id="ocp-4-11-machine-config-operator-bug-fixes"]
 ==== Machine Config Operator
 
+* Because `keyFile` is not configured as the default plug-in for NetworkManager on RHEL nodes, RHEL nodes might not reach the ready state after a reboot.  With this fix, `keyFile is set as the default NetworkManager plug-in on all cluster nodes. As a result, nodes correctly reach the ready state after a reboot. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2060133[*BZ#2060133*])
+
+* Because vSphere UPI clusters do not set the `PlatformStatus.VSphere` parameter at installation, the parameter was set to 'nil`. This caused the MCO logs to be filled with unnecessary and repetitive messages that this parameter cannot be `nil`. This fix removes the warning, which was added to resolve a separate issue. As a result, the logs no longer list this message for vSphere U{I installations. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2080267[*BZ#2080267*])
```

Comment:
```suggestion
* Because vSphere UPI clusters do not set the `PlatformStatus.VSphere` parameter at installation, the parameter was set to `nil`. This caused the MCO logs to be filled with unnecessary and repetitive messages that this parameter cannot have the value `nil`. This fix removes the warning, which was added to resolve a separate issue. As a result, the logs no longer list this message for vSphere UPI installations. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2080267[*BZ#2080267*])
```